src/stories/Library/Modals/ModalShared.tsx

```diff
@@ -28,15 +28,18 @@ export function ModalFallbackButton({
 export function ModalCloseButton({
   toggleModal,
   idAriaDescribedBy,
+  classNames,
 }: {
   toggleModal: void | (() => void);
   idAriaDescribedBy: string;
+  classNames?: string;
 }) {
+  const isOffset = !(classNames && classNames.includes("modal-right"));
```

Comment:
Please make the orientation a prop instead of a "magic" class name which infers other changes.